content/en/apps/tutorials/application-settings.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "CHT Application Settings"
+linkTitle: Application Settings
+weight: 1
+description: >
+  Managing CHT application settings
+relatedContent: >
+  
+---
+
+## Purpose of the Tutorial
+
+This tutorial will take you through how to manage the CHT application settings, including; setting user roles and permissions, enabling and disabling transitions, configuring contact hierarchy and configuring replication.
```

Comment:
this would read better in a bullet point list

Reply:
Updated. Left out the first line on "Settings represent user preferences" since it's probably applicable specifically to user settings vs app settings in general.